pages/index.tsx

```diff
@@ -1,62 +1,54 @@
-import fs from 'fs'
-import path from 'path'
-
-import { useContext } from 'react'
-
-import matter from 'gray-matter'
 import type { NextPage } from 'next'
-import { serialize } from 'next-mdx-remote/serialize'
-import getConfig from 'next/config'
 import Head from 'next/head'
-import { IItemDocs, IGasDocs, IDocMeta } from 'types'
-
-import { EthereumContext } from 'context/ethereumContext'
 
-import ContributeBox from 'components/ContributeBox'
+import Editor from 'components/Editor'
 import HomeLayout from 'components/layouts/Home'
-import ReferenceTable from 'components/Reference'
-import { H1, Container } from 'components/ui'
-
-const { serverRuntimeConfig } = getConfig()
-
-const HomePage = ({
-  opcodeDocs,
-  gasDocs,
-}: {
-  opcodeDocs: IItemDocs
-  gasDocs: IGasDocs
-}) => {
-  const { opcodes } = useContext(EthereumContext)
+import { Container } from 'components/ui'
 
+const HomePage = () => {
   return (
     <>
+      <html lang="en"></html>
       <Head>
         <meta property="og:type" content="website" />
-        <meta property="og:title" content="EVM Codes - Opcodes" />
+        <title>Kakarot Playground - Playground</title>
         <meta
           name="description"
-          content="An Ethereum Virtual Machine Opcodes Interactive Reference"
+          content="Kakarot Playground’ Playground is an interactive tool for learning how to use opcodes, the stack behind smart contracts."
         />
       </Head>
-      <Container>
-        <H1>
-          An Ethereum Virtual Machine <br></br> Opcodes Interactive Reference
-        </H1>
-      </Container>
 
-      <section className="py-10 md:py-20 bg-gray-50 dark:bg-black-700">
-        <Container>
-          <ReferenceTable
-            reference={opcodes}
-            itemDocs={opcodeDocs}
-            gasDocs={gasDocs}
-          />
-        </Container>
-      </section>
+      <Container className="pb-10">
+        <p className="pb-2">
+          Kakarot is an EVM interpreter written in Cairo, a sort of ZK-EVM
```

Comment:
I think we can delete "a sort of ZK-EVM emulator", it does not add information
`Kakarot is a zk-EVM written in Cairo. It leverages STARK proofs to enable provability of execution.`

Reply:
Feel free to reword;p "provability" may sound a bit too much

Reply:
I like provability